content/en/serverless/datadog_lambda_library/_index.md

```diff
@@ -2,6 +2,9 @@
 title: Datadog Lambda Library
 kind: documentation
 further_reading:
+- link: "/serverless/datadog_lambda_library/extension"
+  tag: "Datadog Lambda Extension (Preview)"
```

Comment:
I dunno why all the tags are set to the titles :) they should all be "Documentation"

Reply:
Addressed, thanks!